pyiron/atomistics/job/interactivewrapper.py

```diff
@@ -162,7 +164,11 @@ def db_entry(self):
         """
         db_dict = super(InteractiveWrapper, self).db_entry()
         if self.structure:
-            parent_structure = self.structure.get_parent_basis()
+            if isinstance(self.structure, PAtoms):
+                parent_structure = self.structure.get_parent_basis()
+            else:
+                parent_structure = ase_to_pyiron(self.structure)
```

Comment:
But that is just personal preference: 
```suggestion
                parent_structure = ase_to_pyiron(self.structure).get_parent_basis()
```